### ISSUE TITLE
[minor request] Tickrate Independant Replays

### ISSUE THREAD

**Issue by kidfearless** [shavitush/bhoptimer#737]
It would be a good idea to have replay files include the tickrate it was recorded on. Then have the frames either be added or removed based of the tickrate of the server it's being played on. This would allow replay files to be shared more easily between servers.

**Comment by strafe** [comment]
Why would you share replays across tickrates, a maxed 128 tick route is not possible on 64 tick.

**Comment by kidfearless** [comment]
Yeah, but a 64 tick route is defintely possible on 128 tick, and might as well make it go both ways.
